src/usage/GraphTypeSwitcher.tsx

```diff
@@ -58,10 +61,11 @@ const GraphTypeSwitcher: FC<OwnProps> = ({
       y: {},
     },
     position: 'overlaid',
+    yColumn: usageVector.fluxKey,
```

Comment:
the magic sauce to make the XY line data use the given column data rather than the default `_value`